Microsoft.Toolkit.Uwp.UI.Media/Brushes/AcrylicBrush.cs

```diff
@@ -20,42 +19,42 @@ public sealed class AcrylicBrush : XamlCompositionEffectBrushBase
         /// <summary>
         /// The <see cref="EffectSetter{T}"/> instance in use to set the blur amount
         /// </summary>
-        /// <remarks>This is only set when <see cref="Source"/> is <see cref="AcrylicBackgroundSource.Backdrop"/></remarks>
+        /// <remarks>This is only set when <see cref="BackgroundSource"/> is <see cref="AcrylicBackgroundSource.Backdrop"/></remarks>
         private EffectSetter<float> blurAmountSetter;
 
         /// <summary>
         /// The <see cref="EffectSetter{T}"/> instance in use to set the tint color
         /// </summary>
-        private EffectSetter<Color> tintSetter;
+        private EffectSetter<Color> tintColorSetter;
 
         /// <summary>
         /// The <see cref="EffectSetter{T}"/> instance in use to set the tint mix amount
         /// </summary>
-        private EffectSetter<float> tintMixSetter;
+        private EffectSetter<float> tintOpacitySetter;
 
         /// <summary>
-        /// Gets or sets the source mode for the effect
+        /// Gets or sets the background source mode for the effect (the default is <see cref="AcrylicBackgroundSource.Backdrop"/>).
         /// </summary>
-        public AcrylicBackgroundSource Source
+        public AcrylicBackgroundSource BackgroundSource
```

Comment:
You need to update the AcylicBrush sample with these changes too:
```xml
          <brushes:AcrylicBrush
                                TintColor="DarkCyan"
                                TintOpacity="0.2"
                                BlurAmount="12"
                                TextureUri="ms-appx:///Assets/BrushAssets/NoiseTexture.png"/>
```

Reply:
Whoops! 😅

Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3298/commits/c38640cd61966a05b41a7519aa27ee6fed2f3622.